File: frontend/lib/graphql/queries/company.dart
const String getCompany = r'''
query getCompany($taxId: String){
    getCompany(taxId: $taxId){    
        name,
        vat_number,
        vat,
        registration_number,
        company_address{
            address,
            locality,
            county_code
        }        
    }
}
''';

const String getCompanyByTaxId = r'''
query  getCompanyByTaxId($taxId: String!) {
   getCompanyByTaxId(taxId: $taxId){      
        name,    
        vat_number,
        vat,
        registration_number,
        company_address{
            address,
            locality,
            county_code
        }       
    }
}
''';

File: backend/models/company.go
package models

type Company struct {
	Id                 string   `json:"id"`
	Name               string   `json:"name"`
	VatNumber          *string  `json:"vat_number"`
	Vat                bool     `json:"vat"`
	RegistrationNumber *string  `json:"registration_number"`
	CompanyAddress     *Address `json:"company_address"`
}
